app/components/ProjectList.tsx

```diff
@@ -10,9 +13,42 @@ interface ProjectListProps {
 }
 
 export default function ProjectList(props: ProjectListProps) {
+
+  const [currentProjectIndex, setCurrentProjectIndex] = useState(0);
```

Comment:
Remember to type your useState: `useState<number>(0)`

Reply:
Huh, thanks for shouting that out. I had absolutely no idea that functions could be typed like that.